### PR TITLE
E 882 prevent users to click on the more info links when they

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

File: src/pages/ProjectDetails/Overview/OverviewPanel/index.tsx [hunk 1]
```diff
@@ -106,10 +112,29 @@ export default function OverviewPanel({
       {moreInfoLink && (
         <Section testId="more-info" icon={SectionIcon.Link} title={T("project.details.overview.moreInfo")}>
           <div data-testid="more-info-link" className="flex overflow-hidden text-sm font-semibold text-spacePurple-500">
-            <ExternalLink
-              text={moreInfoName || moreInfoLink.replace(/^https?:\/\//i, "").replace(/\/$/, "")}
-              url={moreInfoLink}
-            />
+            {isLoggedIn ? (
+              <>
```
Comment (kaelsky) [inline src/pages/ProjectDetails/Overview/OverviewPanel/index.tsx:116]: Extra fragment

Comment (haydencleary) [inline src/pages/ProjectDetails/Overview/OverviewPanel/index.tsx:116]: Don't need a fragment here

File: src/pages/ProjectDetails/Overview/OverviewPanel/FakeExternalLink.tsx
[the remark this answers is not in the document]
Comment (alexbeno) [reply src/pages/ProjectDetails/Overview/OverviewPanel/FakeExternalLink.tsx:1]: we don't want to render any link, juste a fake text with icon so it's not the same as the external link components

**Review by kaelsky — APPROVED** [review]
👐

**Pull request merged**